packages/sanity/src/core/hooks/__tests__/useTimeAgo.test.tsx

```diff
@@ -51,22 +51,22 @@ describe('useTimeAgo', () => {
       expect(result.current).toBe('Jun 3, 2022')
     })
 
-    it('outputs eg "3w" if less than a month apart', () => {
+    it('outputs eg "3 wks" if less than a month apart', () => {
```

Comment:
Does this replace the "3w" short format with the slightly longer "3 wks" all over? Since these are used in confined areas of the UI, will this risk running into issues with text no longer fitting the space it's given?

Reply:
It does! @mikolajdobrucki approved this from the design side, but we should still have a look around and see if anything breaks - but we can easily change this after merging if problematic.